CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ After the cluster scoped resource IDs change (ignores namespaces), Kubernetes st
 - Default 5 minute timeout for any apply operation.
 - `--execution-timeout` flag for the apply method to override the default timeout
 - `--logger` flag to set the logger type, available optios are: default, json and simple.
-- `--delete-first` flag that inverts the actions order on resources, `delete` before `apply`.
+- By default now Kahoy will first delete and then apply. We find this way of execution safer than applying and then deleting.
```

Comment:
This point better under changes

Reply:
Addressed in https://github.com/slok/kahoy/pull/186/commits/78d4f430105355fa7fa24f4f0ec0473b23e66310